tmt/steps/discover/fmf.py

```diff
@@ -557,6 +557,11 @@ def assert_git_url(plan_name: Optional[str] = None) -> None:
                 'modified-ref',
                 tmt.utils.default_branch(repository=self.testdir, logger=self._logger))
             self.info('modified-ref', modified_ref, 'green')
+            ref_commit = self.run(
+                Command('git', 'rev-parse', '--short', str(modified_ref)),
+                cwd=self.testdir)
+            assert ref_commit.stdout is not None
+            self.debug(modified_ref, ref_commit.stdout.strip())
```

Comment:
Hmm, isn't tmt already logging the output of this command? Both `stdout` and `stderr`?

Reply:
It is. So the question is whether make that line  verbose or delete (and above assert as well).
Currently one needs to see `log.txt` and there is this:
```
        modified-ref: reference/master
        Run command: git rev-parse --short reference/master
        out: 03011d2
        reference/master: 03011d2
```

Reply:
changed in 6218b1a0. thanks for the comment